gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy

```diff
@@ -154,15 +155,12 @@ interface ClouddriverService {
                        @Query("cloudProvider") String cloudProvider)
 
   @Headers("Accept: application/json")
-  @GET("/applications/{name}/jobs")
-  List getJobs(@Path("name") String name, @Query("expand") String expand)
-
-  @Headers("Accept: application/json")
-  @GET("/applications/{name}/jobs/{account}/{region}/{jobName}")
+  @POST("/applications/{name}/jobs/{account}/{region}/{jobName}")
```

Comment:
Ideally, probably change clouddriver to be a get, but `[orca` already depends on this being `POST`](https://github.com/spinnaker/orca/blob/7c4896117fdcd6930cadaa82bfdf5dcb8383f89b/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/KatoRestService.groovy#L47) and seems like not a great idea to make a breaking change like that